style.css

```diff
@@ -0,0 +1,300 @@
+:root {
+    font-size: 62.5%;
+}
+
+*, *::after, *::before {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+html, body {
+    font-family: 'Roboto', sans-serif;
+    height: 100%;
+}
+
+.app {
+    position: relative;
+    background-color: #D4F0F0;
+    min-height: 100%;
+}
+
+h1 {
+    font-family: 'Caprasimo', sans-serif;
+    font-size: 3rem;
+    text-align: center;
+    padding: 2rem;
+    color: white;
+    background-color: #8FCACA;
+}
+
+.flex_row {
+    display: flex;
+    flex-direction: row;
+}
+
+.flex_column {
+    display: flex;
+    flex-direction: column;
+}
+
+/* ADD A TASK FORM */
+
+#new_task_form {
+    margin-top: 5rem;
+    margin-bottom: 5rem;
+    margin-left: 21rem;
+    gap: 0.5rem;
+}
+
+
+#new_task_text {
+    font-weight: 600;
+    width: 30rem;
+    padding: 1rem;
+    border: none;
+    border-radius: 1rem;
+    color: white;
+    background-color: #FFAEA5;
+}
+
+#select_progress {
+    font-weight: 600;
+    padding: 1rem;
+    border: none;
+    border-radius: 1rem;
+    color: white;
+    background-color: #FFDBCC;
+}
+
+#new_task_btn {
+    font-size: 2rem;
+    font-weight: 600;
+    width: 4rem;
+    border: none;
+    background-color: white;
+    border-radius: 1rem;
+    cursor: pointer;
+}
+
+
+/* LIST CONTAINER */
+
+.list_container {
+    gap: 1rem;
+}
+
+.list_column {
+    width: 100%;
+    height: 100%;
+    margin: 2rem;
+}
+
+.tasks_container{
+    min-height: 30vh;
+}
+
+.title {
+    height: 4rem;
+    align-items: center;
+    justify-content: space-between;
+    padding-inline: 2rem 1rem;
+    border-radius: 1rem;
+    background-color: #FFC8A2;    
+}
+
+.title h2 {
+    font-size: 2rem;
+    font-weight: 600;
+    color: white;
+}
+
+.sort_btn {
+    height: 3rem;
+    width: 4rem;
+    margin-right: 0.5rem;
+    background-image: url(./resources/sort.png);
```

Comment:
I think it would be better to use images in semantic <img /> tag with proper alt values